src/cmd/connect.go

```diff
@@ -22,83 +22,80 @@ var (
 	connectLocalPort    int
 	connectRemotePort   int
 	cliOnly             bool
+)
 
-	connectCmd = &cobra.Command{
-		Use:     "connect { REGISTRY | GIT | connect-name }",
-		Aliases: []string{"c"},
-		Short:   lang.CmdConnectShort,
-		Long:    lang.CmdConnectLong,
-		RunE: func(cmd *cobra.Command, args []string) error {
-			var target string
-			if len(args) > 0 {
-				target = args[0]
-			}
-			spinner := message.NewProgressSpinner(lang.CmdConnectPreparingTunnel, target)
-			defer spinner.Stop()
-			c, err := cluster.NewCluster()
-			if err != nil {
-				return err
-			}
+var connectCmd = &cobra.Command{
+	Use:     "connect { REGISTRY | GIT | connect-name }",
+	Aliases: []string{"c"},
+	Short:   lang.CmdConnectShort,
+	Long:    lang.CmdConnectLong,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		target := ""
+		if len(args) > 0 {
+			target = args[0]
+		}
 
-			ctx := cmd.Context()
+		spinner := message.NewProgressSpinner(lang.CmdConnectPreparingTunnel, target)
+		defer spinner.Stop()
 
-			var tunnel *cluster.Tunnel
-			if connectResourceName != "" {
-				zt := cluster.NewTunnelInfo(connectNamespace, connectResourceType, connectResourceName, "", connectLocalPort, connectRemotePort)
-				tunnel, err = c.ConnectTunnelInfo(ctx, zt)
-			} else {
-				tunnel, err = c.Connect(ctx, target)
-			}
-			if err != nil {
-				return fmt.Errorf("unable to connect to the service: %w", err)
-			}
+		c, err := cluster.NewCluster()
+		if err != nil {
+			return err
+		}
 
-			defer tunnel.Close()
-			url := tunnel.FullURL()
+		ctx := cmd.Context()
 
-			// Dump the tunnel URL to the console for other tools to use.
-			fmt.Print(url)
+		var tunnel *cluster.Tunnel
+		if connectResourceName != "" {
```

Comment:
would swap from a `!= ""` to a `= ""` 
Thought there would be an `isEmpty()` in the strings lib, but there isn't :(